.hlint.yaml

```diff
@@ -0,0 +1,2 @@
+- ignore: {name: Redundant <$>, within: [Hydra.Contract.HeadTokens]}
```

Comment:
Should: Leave a hint on the corresponding code location

You mentioned that we can't have a local ignore pragma because of plutus-tx, but can we somehow make a link that we want to get rid of that change the next time when we recompile / change the script hashes?